File: clients/dotnet/enums/DocumentType.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Net.Http;
using System.Text;
using System.Threading.Tasks;
using Newtonsoft.Json;

namespace Avalara.AvaTax.RestClient
{
    /// <summary>
    /// 
    /// </summary>
    public enum DocumentType
    {
        /// <summary>
        /// 
        /// </summary>
        SalesOrder,

        /// <summary>
        /// 
        /// </summary>
        SalesInvoice,

        /// <summary>
        /// 
        /// </summary>
        PurchaseOrder,

        /// <summary>
        /// 
        /// </summary>
        PurchaseInvoice,

        /// <summary>
        /// 
        /// </summary>
        ReturnOrder,

        /// <summary>
        /// 
        /// </summary>
        ReturnInvoice,

        /// <summary>
        /// 
        /// </summary>
        InventoryTransferOrder,

        /// <summary>
        /// 
        /// </summary>
        InventoryTransferInvoice,

        /// <summary>
        /// 
        /// </summary>
        ReverseChargeOrder,

        /// <summary>
        /// 
        /// </summary>
        ReverseChargeInvoice,

        /// <summary>
        /// 
        /// </summary>
        Any,


    }
}

File: clients/php/enums/DocumentType.php
<?php
class DocumentType extends AvaTaxEnum 
{

    const SalesOrder = "SalesOrder";
    const SalesInvoice = "SalesInvoice";
    const PurchaseOrder = "PurchaseOrder";
    const PurchaseInvoice = "PurchaseInvoice";
    const ReturnOrder = "ReturnOrder";
    const ReturnInvoice = "ReturnInvoice";
    const InventoryTransferOrder = "InventoryTransferOrder";
    const InventoryTransferInvoice = "InventoryTransferInvoice";
    const ReverseChargeOrder = "ReverseChargeOrder";
    const ReverseChargeInvoice = "ReverseChargeInvoice";
    const Any = "Any";
}
?>
